src/pipeline/broad_phase/broad_phase.rs

```diff
@@ -41,12 +69,20 @@ pub trait BroadPhase<N: Real, BV, T>: Any + Sync + Send {
     /// Forces the broad-phase to recompute and re-report all the proximities.
     fn deferred_recompute_all_proximities(&mut self);
 
+    /// Updates the object additions, removals, and interferences detection.
+    fn update_with_handler(&mut self, handler: &mut BroadPhaseProximityHandler<T>);
```

Comment:
This should just replace the existing `update` method to avoid having several ways of doing the same thing.